avocado_vt/loader.py

```diff
@@ -107,7 +108,10 @@ def _get_parser(self):
 
     def get_extra_listing(self):
         if self.args.vt_list_guests:
-            guest_listing(self.args)
+            args = copy.copy(self.args)
+            args.vt_config = None
+            args.vt_guest_os = None
```

Comment:
Well I'm also inclined to that, let me explore this option in v2 (no promises as it means yet another version of argparse options, because it supports only a subset of the options, so `vt-list-guest` should not list them)